apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.ts

```diff
@@ -204,10 +205,19 @@ export const getIndictmentDefendantsInfo = (theCase: Case) => {
       ? Date.now() >= new Date(verdictAppealDeadline).getTime()
       : false
 
+    const sentToPrisonAdminDate = theCase.defendantEventLogs?.find(
+      (defendantEventLog) =>
+        defendantEventLog.eventType ===
+          DefendantEventType.SENT_TO_PRISON_ADMIN &&
+        defendantEventLog.caseId === theCase.id &&
+        defendantEventLog.defendantId === defendant.id,
+    )?.created
```

Comment:
@gudjong @unakb, we need to make sure that we find the newest entry, since we don't remove from the event log on revocation. Is this good enough or should I sort the filtered list here?

Reply:
We could also just order them in the query itself like we do for EventLogs, but it's true we should order them somewhere at least so we get the correct one

Reply:
I suggest ordering them server side. You don´t need to compare the case ids as you only get logs that belong to the current case. Also, you don't need the case id in the database table, it´s enough to link to the defendant table.

Reply:
I'll sort the logs server side 🤝

I initially thought the caseId needed to be present as well since we wanted to be able to get logs per defendant in a given case, but I now realise that if the same defendant exists in different cases, they would get another id in those cases. 

I'll remove the case id.

Reply:
I've now sorted the logs server side. I'd like some help with removing the `caseId` if someone could. There are other comments regarding that so I'll close this convo.